src/index.ts

```diff
@@ -44,6 +46,7 @@ declare global {
     _fs_cookie_domain?: string;
     _fs_debug?: boolean;
     _fs_dev_mode?: boolean;
+    _fs_disable_logs?: boolean;
```

Comment:
Since the flag only controls the logging inside the SDK, maybe just creating the `disableLogs` flag could achieve the goal. The global flag `_fs_disable_logs` may be omitted. 
something like
```
// local var to browser SDK only, can be mutated by init options
let _fsDisableSdkLogs = false;
const log = (message: string) => {
  if (!_fsDisableSdkLogs) {
    console.warn(message);
  }
};
```